### PR TITLE
bring back the previous readme version

### PR DESCRIPTION
**Pull request by AgaK-O**
Bring back the accidentally changed part of readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,22 +17,36 @@ deploy changes with:
...
 
-run locally:
+1. Fork this repository on GitHub.
+2. Clone *your forked repository* (not our original one) to your hard drive with `git clone https://github.com/YOURUSERNAME/slate.git`
+3. `cd slate`
+4. Initialize and start Slate. You can either do this locally, or with Vagrant:
 
 ```shell
-./run.sh
+# either run this to run locally
```
Comment (chmurson) [inline README.md]: but we want to use `./run.sh` to execute it right ?

File: README.md [hunk 2]
```diff
@@ -17,22 +17,36 @@ deploy changes with:
...
 ```
 
-You can now see the docs at http://localhost:4567. 
+You can now see the docs at http://localhost:4567. Whoa! That was fast!
 
-deploy changes with:
+Now that Slate is all set up on your machine, you'll probably want to learn more about [editing Slate markdown](https://github.com/lord/slate/wiki/Markdown-Syntax), or [how to publish your docs](https://github.com/lord/slate/wiki/Deploying-Slate).
 
-```shell 
-./deploy.sh
```
Comment (chmurson) [inline README.md]: You remove here deployment instructions, but this is important to have them. Or isn't it ?

File: README.md (hunk 1 above)
Comment (AgaK-O) [reply README.md]: It is already at the top of readme.md file. This was changed by accident.

File: README.md (hunk 2 above)
Comment (AgaK-O) [reply README.md]: It is already at the top of readme.md file. This was changed by accident.

**Pull request merged**